modules/google/stt.py

```
@@ -0,0 +1,291 @@
+# modules/google/stt.py
+import dataclasses
+import queue
+import threading
+import time
```

Ruff (0.11.9)

`time` imported but unused

Remove unused import: `time`

(F401)